agent/exec/controller.go

```diff
@@ -183,6 +190,8 @@ func Do(ctx context.Context, task *api.Task, ctlr Controller) (*api.TaskStatus,
 			panic("err must not be nil when fatal")
 		}
 
+		span.SetTag("error", err)
```

Comment:
`"error"` is "special"... per https://github.com/opentracing/specification/blob/master/data_conventions.yaml#L16 , it is expected to be a `bool`. I'm actually debating the finer points of more detailed error information right now in an open PR: https://github.com/opentracing/specification/pull/35